controllers/front/ProductController.php

```diff
@@ -954,6 +960,29 @@ protected function formatQuantityDiscounts($specific_prices, $price, $tax_rate,
         return $specific_prices;
     }
 
+    /**
+     * @param float $specificPriceValue
+     * @param bool $specificPriceReductionTax
+     * @param float $ecoTaxAmount
+     *
+     * @return mixed
+     */
+    private function calculatePriceDefaultCurrency(
```

Comment:
```suggestion
    private function calculatePriceDefaultCurrencyAmount(
```

You're not calculating a currency. :thinking: 
Wdyt @matks?

Reply:
It only calculate a price, in default currency because we take data from a specific price without applying another currency.

I gave this name according to the variable it fills in the parent function, but you are right, this function has no knowledge of any currency matters. I think I should change that.

Reply:
I called it calculateSpecificPrice, because in the end this is what it really does, for a specific price basis, it calculate its final value (with or without tax rate and ecotax)